draft-ietf-taps-interface.md

```diff
@@ -595,6 +597,8 @@ for Protocol-specific Properties and MUST NOT be used for vendor or implementati
 Terms listed as keywords as in the protocol numbers registry SHOULD be avoided as any part of a vendor- or
 implementation-specific property name.
 
+Though Transport Property Names are case-insensitive, it is recommended to use camelCase to improve readability.
+Implementations may transpose Transport Property Names into snake_case or PascalCase to blend into the language environment.
```

Comment:
I note that this is pretty strict... we don't know how styles of future languages will be?  But I like it nevertheless. If we allow "any" kind of transposition, we make porting applications harder.